### ISSUE TITLE
Share screen not working on Firefox 60.0.2

### ISSUE THREAD

**Issue by miguelrcDEV** [OpenVidu/openvidu#71]
I init the publisher like: 
```
let sharedScreen = OV.initPublisher('', 
        { 
              videoSource: "screen", 
              publishAudio: false 
        }, 
        (error) => {
               if (error.name === 'SCREEN_EXTENSION_NOT_INSTALLED') {
                        alert(error.message);
               } else if (error.name === 'SCREEN_EXTENSION_DISABLED') {
                        alert('You need to enable screen sharing extension');
               } else if (error.name === 'SCREEN_CAPTURE_DENIED') {
                        alert('You need to choose a window or application to share');
               }
         }
);
```
But Firefox can't show share screen. Could you check this please? Maybe its an issue in openvidu-browser

**Comment by pabloFuente** [comment]
What kind of error output can you see?

I've just tried screen-sharing on Firefox 60.0.2 and it works OK. The browser asks the user to select a window to capture and the video is properly sent. A little more information, please?

**Comment by miguelrcDEV** [comment]
Solved!
The problem was I initPublisher like:
```
let sharedScreen = OV.initPublisher('', 
{ 
      videoSource: "screen"
}
```

But to make work in firefox I had to put `audioSource: false` like:
```
let sharedScreen = OV.initPublisher('',
{
      videoSource: "screen",
      audioSource: false
}
```

**Comment by pabloFuente** [comment]
Is there something particular about your computer? (maybe no audio device input or something like that)

Firefox 60.0.2, initializing the Publisher like this:

```
let sharedScreen = OV.initPublisher('',
{
      videoSource: "screen",
      audioSource: undefined
}
```

Got asked for screen capturing permissions first, and for microphone permissions just after that. Works as expected (remote user receives both the shared window and the audio)

**Comment by miguelrcDEV** [comment]
Yep, but when in firefox has a normal publisher using the same audioDevice as the share screen wants to use, throw the error, so I need to put audioSource to false. Don't worry about this, because for me its the expected behaviour: share screen without audio.

**Comment by pabloFuente** [comment]
Okey, but doing some more tests I cannot reproduce your error in any way. At least in my machine, only the webcam cannot be accesed by Chrome and Firefox at the same time. There's no problem at all when accessing the microphone, and for the same browser (connecting publishers to one session in the same tab or in different tabs, doesn't matter) it never throws a conflict error, neither for the camera nor for the microphone.

To sum up, I am able to start multiple publishers in Firefox sharing the webcam or the screen and all of them also accessing the michrophone. And I can also start at the same time publishers in Chrome, accessing both the screen and the microphone (not the webcam).

So the error you had may be something related to your particular hardware configuration, I guess.

Regards.

**Comment by miguelrcDEV** [comment]
Yes, when I have more time to debug and try, I'll keep an eye on it, to see what the error may be ;) . Anyway, thank you very much for the work you do and for the support you give, especially to you Pablo. I do not know if you have thought about creating a group of slack in which to be able to support and comment things among everyone, it would be great.

**Comment by kebiromar** [comment]
Hello, to share the screen and the audio in the same time you can  use the same process for sharing cam  and change the value of the variable screen to true 
`     let publisher = this.OV.initPublisher('', {
          audio: true,        // Whether you want to transmit audio or not
          video: true,        // Whether you want to transmit video or not
          audioActive: true,  // Whether you want to start the publishing with your audio unmuted or muted
          videoActive: true,  // Whether you want to start the publishing with your video enabled or disabled
          quality: 'MEDIUM',  // The quality of your video ('LOW', 'MEDIUM', 'HIGH')
          screen: true       // true to get your screen as video source instead of your camera
        });`

**Comment by kebiromar** [comment]
sorry ,
     let publisher = this.OV.initPublisher('', {
          audio: true,        // Whether you want to transmit audio or not
          video: true,        // Whether you want to transmit video or not
          audioActive: true,  // Whether you want to start the publishing with your audio unmuted or muted
          videoActive: true,  // Whether you want to start the publishing with your video enabled or disabled
          quality: 'MEDIUM',  // The quality of your video ('LOW', 'MEDIUM', 'HIGH')
          screen: true       // true to get your screen as video source instead of your camera
        });
